components/tensorflow-notebook-image/Dockerfile

```diff
@@ -206,6 +206,9 @@ RUN conda_packages=$(conda list -e | cut -d '=' -f 1 | grep -v '#' | sort) && \
     python -m ipykernel install --user && \
     echo "${pip_only_packages}" | xargs -n 1 -I "{}" /bin/bash -c 'pip install --no-cache-dir {} || true' && \
     pip install --no-cache-dir tensorflow-transform && \
+    pip install --no-cache-dir tensorflow-model-analysis && \
+    jupyter nbextension install --py --symlink tensorflow_model_analysis --system && \
```

Comment:
Why --symlink?

Reply:
The reason I included it was that I started with the instructions from the model-analysis README which uses `--symlink`, and that part doesn't cause issues in our build (the default install location did cause issues, but that is orthogonal).

From what I understand, passing in `--symlink` is considered a best-practice since it saves space on non-Windows systems. However, I don't have a strong preference one way or the other.